scripts/tests/api_compare/filter-list-offline

```diff
@@ -8,6 +8,7 @@
 !Filecoin.NetAgentVersion
 !Filecoin.NetAutoNatStatus
 !Filecoin.NetPeers
+!Filecoin.NetFindPeer
```

Comment:
Where is this method invoked in the tests?

Reply:
Not invoked at this moment but it's not available on an offline server

Reply:
Why is it not invoked?

Reply:
With the current testing infra, it's hard to automate the testing for this one

Reply:
Why?

Reply:
Can we do at least "unhappy" path test? That is, to find a peer that we _know_ doesn't exist?

Reply:
`.ignore("It times out in lotus when peer not found")`